README.md

```diff
@@ -24,7 +40,28 @@ npm install
 yarn add
 ```
 
-Third step, run the development server:
+If you face any issue installing the project dependencies, use this command.
+
+```bash
+npm install --f
+# or
+yarn install --force
+```
+
+## Setting up Development Server for testnet 🌴 
+
+- In the root of your project, create a `.env.local` file.
+- Copy all the content in `.env.test` into your newly created file
```

Comment:
.env.test is incomplete. I think you can just do one part asking to copy .env.example to .env.local.

Reply:
I guess I can use the notion document you sent to me and complete it, because I feel that's the essence of the read me actually. So people can use testnet more for development purposes. What do you think?